core/src/main/java/org/commonjava/maven/ext/core/state/RESTState.java

```diff
@@ -84,6 +92,8 @@ public void initialise( Properties userProps )
         restSuffixAlign = Boolean.parseBoolean( userProps.getProperty( REST_SUFFIX, "true" ) );
 
         String repositoryGroup = userProps.getProperty( REST_REPO_GROUP, "" );
+        Boolean brewPullActive = BooleanUtils.toBooleanObject( userProps.getProperty( REST_BREW_PULL_ACTIVE ) );
```

Comment:
Why can't we use `Boolean.parseBoolean` here?

Reply:
Boolean.parseBoolean produces boolean, not Boolean, i.e. the null value (when not provided on command line) gets lost in favor of a default one. But the default should be decided on DA side, not in PME.

Reply:
Won't it get serialized via jackson when calling DA? So it would end up as false/true ?

Reply:
It depends on the serialization implementation, but I expected that null value will be skipped from putting into the resulting json.
But anyway, the exptra fields are ignored and in DA 2.1 these become mandatory, so sending in a value is probably ok. Will update it to use `Boolean.parseBoolean()`.